tests/testthat/test-navbar.R

```diff
@@ -110,7 +110,7 @@ test_that("data_navbar() can remove elements", {
         right: ~
   ")
 
-  expect_snapshot(data_navbar(pkg))
+  expect_equal(data_navbar(pkg)$right, "")
```

Comment:
Replaced with an equality test since we know what we expect here, and it's simple.